src/worker/event/lambdaServer.go

```diff
@@ -16,6 +20,29 @@ type LambdaServer struct {
 	lambdaMgr *lambda.LambdaMgr
 }
 
+// Converts a kafka message to a HTTP request, since the LambdaServer Invocation and everything else relies on
```

Comment:
We should do some refactoring of the code in the events directory.  Each event type should be its own file (at this point, one for HTTP and one for Kafka).